WearStandaloneGoogleSignIn/app/build.gradle

```diff
@@ -1,15 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
-
-    buildToolsVersion "28.0.3"
+    compileSdkVersion 29
 
     defaultConfig {
         applicationId "com.example.android.wearable.wear.wearstandalonegooglesignin"
 
         minSdkVersion 25
-        targetSdkVersion 28
+        targetSdkVersion 29
```

Comment:
Still want to target 28 as Wear OS hasn't been released yet with Q (29).

Reply:
Fixed in 5c2e340